config.toml

```diff
@@ -80,6 +80,12 @@ enableInlineShortcodes = true
     parent = "Faith Formation"
     weight = 3
 
+  [[menu.main]]
+    name = "CCD Team"
+    url = "/ccd-team"
+    parent = "Faith Formation"
+    weight = 1
```

Comment:
weight would be 2 here and the weights of other menu items below would be 3 and 4.